membership-lockdown.php

```diff
@@ -48,8 +48,7 @@
 	register_activation_hook( __FILE__, function() {
 			$lockdown_status = 0;
 			update_option( 'mlockdown_status', $lockdown_status );
-		}
-	);
+	});
```

Comment:
:no_entry_sign: **Error**: Closing parenthesis of a multi-line function call must be on a line by itself (*PEAR.Functions.FunctionCallSignature.CloseBracketLine*).